variants/cell-328pb/pins_arduino.h

```diff
@@ -102,6 +102,11 @@ static const uint8_t A5 = PIN_A5;
 static const uint8_t A6 = PIN_A6;
 static const uint8_t A7 = PIN_A7;
 
+#define PIN_VIVIWARE_EN_RX     (7)
+#define PIN_VIVIWARE_EN_TX     (8)
+#define PIN_VIVIWARE_EN_PWR    (PIN_A3)
```

Comment:
@dodo5522 質問ですが、この定義は `A3` とすると何か問題がありますか？
単純によく使う（よく見る）使い方は `PIN_A3` ではなく `A3` な気がしたので…

```suggestion
#define PIN_VIVIWARE_EN_PWR    (A3)
```

Reply:
@TomohiroHayashi `A3`でも特に問題はないんですが、pins_arduino.hは直接目にすることが少ないので、プリプロセス処理で統一しておこうかな…くらいの意図でした。

ただ、よく考えると、`A0`~`A7`をわざわざコンパイラに処理させるために`uint8_t`宣言しているのは、`pinMode`や`analogWrite`のピン引数を型チェックするためとも思えるので、なるべく安全側に倒すという意味でも`A3`が良さそうです。